app/containers/MembersSection/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { CompositeNavigationProp, useNavigation } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React from 'react';
+import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { setLoading } from '../../actions/selectedUsers';
+import { TSubscriptionModel } from '../../definitions';
+import i18n from '../../i18n';
+import { useUserPermissions } from '../../lib/hooks/useUserPermissions';
+import log, { events, logEvent } from '../../lib/methods/helpers/log';
+import { Services } from '../../lib/services';
+import { MasterDetailInsideStackParamList } from '../../stacks/MasterDetailStack/types';
+import { ChatsStackParamList } from '../../stacks/types';
+import * as List from '../List';
+
+type TNavigation = CompositeNavigationProp<
+	StackNavigationProp<ChatsStackParamList, 'RoomActionsView'>,
+	StackNavigationProp<MasterDetailInsideStackParamList>
+>;
+
+interface IMembersSection {
+	rid: TSubscriptionModel['rid'];
+	t: TSubscriptionModel['t'];
+	joined: boolean;
+}
+
+export default function MembersSection({ rid, t, joined }: IMembersSection): React.ReactElement {
```

Comment:
I can't visualize where we would be using this component outside of RoomMembersView.
Can we colocate it closer to the component where it's going to be used?